include/itkNonLocalPatchBasedImageFilter.h

```diff
@@ -78,13 +104,13 @@ class ITK_TEMPLATE_EXPORT NonLocalPatchBasedImageFilter :
   using NeighborhoodOffsetType = typename ConstNeighborhoodIteratorType::OffsetType;
 
   using NeighborhoodOffsetListType = std::vector<NeighborhoodOffsetType>;
-  /**
-   * Neighborhood patch similarity metric enumerated type
-   */
-  enum SimilarityMetricType {
-    PEARSON_CORRELATION,
-    MEAN_SQUARES
-  };
+
+  using SimilarityMetricTypeEnum = NonLocalPatchBasedImageFilterEnums::SimilarityMetric;
```

Comment:
should this be `SimilarityMetricEnum` (without `Type` in the name)?

Reply:
:eyes: Agreed ! Thanks for pointing this out.  Done in bf8a591.